proposed/authelia/latest/templates/authelia-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: authelia-config
+data:
+  config.yml: |
+    port: 80
+    logs_level: {{ .Values.authelia.config.logsLevel }}
+    {{- if .Values.authelia.ingress.hosts }}
+    default_redirection_url: {{ index .Values.authelia.ingress.hosts 0 }}
+    {{- end }}
+    authentication_backend:
+      {{- if eq .Values.authelia.config.authentication_backend "ldap" }}
+      {{- if .Values.authelia.config.ldap }}
+      ldap:
+{{ toYaml .Values.authelia.config.ldap | indent 8 }}
+      {{- end }}
+      {{- else }}
+      file:
+        path: /authelia/users_database.yml
+      {{- end }}
+    authentication_methods:
+      default_method: single_factor
+    access_control:
+      default_policy: deny
+      any:
+        - domain: {{- printf " '*.%s'" .Values.authelia.config.domain }}
+          policy: deny
+      groups: {}
+      users: {}
+    session:
+      secret: {{ .Values.authelia.config.session.secret }}
```

Comment:
questions.yml `#22`: `secret will be auto-generated if not specified` should either be changed to `required=true` or add auto generate `{{ randAlphaNum 10 | b64enc | quote }}` in here.

Reply:
I prefer not to auto-generate in the helm chart due to the issue brought up here: https://github.com/helm/helm/issues/3053. I prefer UI generating a random value for me.

Reply:
The session secret specified here is part of a config file and I don't have control over the way application specifies this secret. Will try to file an upstream issue.